examples/src/integrationTest/java/org/apache/ignite/example/storage/ItPersistentPageMemoryStorageExampleTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.example.storage;
+
+import static org.apache.ignite.example.ExampleTestUtils.assertConsoleOutputContains;
+
+import java.util.concurrent.TimeUnit;
+import org.apache.ignite.example.AbstractExamplesTest;
+import org.apache.ignite.internal.storage.pagememory.configuration.schema.PageMemoryStorageEngineConfiguration;
+import org.junit.jupiter.api.Test;
+
+/**
+ * For {@link PersistentPageMemoryStorageExample} testing.
+ */
+public class ItPersistentPageMemoryStorageExampleTest extends AbstractExamplesTest {
```

Comment:
Why do we need a separate class for this? Can we use a single class with two methods? We could also extract the configuration into a helper method

Reply:
Makes sense